Source/CombatExtended/CombatExtended/ExplosionCE.cs

```diff
@@ -403,26 +403,26 @@ public override void ExposeData()
         }
 
         //New methods
-        public int GetDamageAmountAtCE(IntVec3 c)   //t => t^(0.333f)
```

Comment:
Everything suggested between 406 and 425 is just screwing with the physics involved.